client/src/static/gromets/SIRD_gromet_PetriNetClassic_metadata.json

```diff
@@ -195,7 +195,7 @@
     {
       "provenance": {
         "method": "FromPNC_baas@gt",
-        "timestamp": "2021-07-07T15:27:484320_UTC-0000",
+        "timestamp": "2021-07-07T15:27:48:4320_UTC-0000",
```

Comment:
This will then probably need to be corrected in donu side since we will be moving to ingesting the models from them?

Reply:
Nope, that was an issue on the GroMET file for some reason.
As far as I worked with Donu API they send the date in the proper format.
If this issue comes back, it's a simple fix.